README.md

```diff
@@ -1,3 +1,11 @@
-#PPE DJANGO
-
-Project oriented to create a customers friendly environment to serve mailing and packages necessities from Puerto Plata, Dominican Republic to the world.
+#PPE DJANGO
```

Comment:
I think you  could do this better, if you want there is a short course on Udacity for writing READMES.